### ISSUE TITLE
Roboflow YOLOv5 Custom Training Notebook

### ISSUE THREAD

**Issue by Jacobsolawetz** [roboflow-ai/roboflow-python#15]
https://colab.research.google.com/drive/1SQ_nlPiovGTGc2IfWmf9EIyNxCne4f3U

Threading here for feedback

**Comment by yeldarby** [comment]
> Roboflow Custom Training with YOLOv5
I don't think we want Roboflow in the title; ideally this is the way we're framing to custom train YOLOv5 period.

> In this tutorial, we assemble a custom dataset and train a custom YOLOv5 model to recognize our custom objects.
That's a lot of customs.

> !pip install roboflow
It would be ideal if Glenn just added us to yolov5's requirements.txt file instead of having this be a separate line.

> rf = Roboflow(notebook="ultralytics")
Needs model="yolov5" as well; this is what sets the export flow -- we want to be able to use this in other notebooks that don't trace back to `ultralytics`' referral link.

> os.environ["DATASET_DIRECTORY"] = "/content/datasets"
This should go up in the setup cell otherwise they're going to copy/paste over the top of it.

> !python train.py --img 416 --batch 16 --epochs 150 --data {dataset.location}/data.yaml --weights yolov5s.pt --cache
Image size should be pulled from the Roboflow info -- we know the resize they used. We should return it as part of the `dataset` object and pass it through like the yaml location.

> See this guide for model performance improvement.
Not linked.

> To deploy your model to an application, see this guide on exporting your model to deployment destinations.
Also not linked.

---

**Other Stuff**

* Stripping wandb from this notebook is poor form imo. We also don't want to give Glenn an argument that our stuff doesn't belong in future notebooks because wandb isn't in ours.
* Next steps could include a bit about the upload API for closing the loop even though it won't be automated as part of inference.

**Comment by Jacobsolawetz** [comment]
Here's the most recent copy incorporating feedback https://colab.research.google.com/drive/1SQ_nlPiovGTGc2IfWmf9EIyNxCne4f3U

**Comment by yeldarby** [comment]
This still needs the `format=` piece
![image](https://user-images.githubusercontent.com/870796/135022709-f7134535-7afb-41ca-b8c6-bf83a45a7210.png)

Model size still not automatically set via variable
`!python train.py --img 416`

Still think an upload api snippet at the end would be a really good hook to put active learning in peoples' mind.
